### PR TITLE
[stable/rabbitmq] Add NetworkPolicy simple support.

### PR DESCRIPTION
**Pull request by desaintmartin**
#### Special notes for your reviewer:

By default, only a single rule for all ports. This may be too permissive, but it is a start.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/rabbitmq/values-production.yaml [hunk 1]
```diff
@@ -265,6 +265,31 @@ resources:
     memory: 256Mi
     cpu: 100m
 
+
```
Comment (carrodher) [inline stable/rabbitmq/values-production.yaml]: There is an unnecessary line that doesn't appear in the _values.yaml_

**Review by carrodher — APPROVED** [review]
Thank you for your contribution, changes LGTM. Just a small comment

File: stable/rabbitmq/templates/networkpolicy.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{{- if .Values.networkPolicy.enabled }}
+kind: NetworkPolicy
+apiVersion: {{ template "rabbitmq.networkPolicy.apiVersion" . }}
```
Comment (carrodher) [inline stable/rabbitmq/templates/networkpolicy.yaml]: Where is defined this `rabbitmq.networkPolicy.apiVersion`>

Comment (desaintmartin) [reply stable/rabbitmq/templates/networkpolicy.yaml]: Indeed, fixed by hardcoding it (since it has been out since kube 1.7)

Comment (desaintmartin) [reply stable/rabbitmq/templates/networkpolicy.yaml]: Indeed, fixed by hardcoding it (since it has been out since kube 1.7)

**Pull request merged**